api/tests/routes/backoffice/collective_offer_templates_test.py

```diff
@@ -231,6 +232,29 @@ def test_list_collective_offer_templates_by_format(self, authenticated_client):
         assert rows[0]["ID"] == str(target_offer.id)
 
 
+class GetCollectiveOfferTemplateDetailTest(GetEndpointHelper):
+    endpoint = "backoffice_web.collective_offer_template.get_collective_offer_template_details"
+    endpoint_kwargs = {"collective_offer_template_id": 1}
+    needed_permission = perm_models.Permissions.READ_OFFERS
+
+    def test_nominal(self, authenticated_client):
+        collectiveOfferTemplate = educational_factories.CollectiveOfferTemplateFactory(
+            formats=[EacFormat.PROJECTION_AUDIOVISUELLE],
+        )
+        url = url_for(self.endpoint, collective_offer_template_id=collectiveOfferTemplate.id)
+        with assert_num_queries(3):
```

Comment:
Bonne pratique : dans la plupart des tests, on définit un attribyt `expected_num_queries`, donnant en commentaire la raison du nombre, et d'utiliser ici : 
```suggestion
        with assert_num_queries(self.expected_num_queries):
```
Ainsi tu définis une seul fois le 3 pour plusieurs tests.